atlassian_jwt_auth/contrib/aiohttp/__init__.py

```diff
@@ -2,8 +2,15 @@
 import sys
 
 if sys.version_info >= (3, 5):
-    from .auth import JWTAuth
-    from .key import HTTPSPublicKeyRetriever
-    from .verifier import JWTAuthVerifier
+    try:
+        import aiohttp
+        from .auth import JWTAuth
+        from .key import HTTPSPublicKeyRetriever
+        from .verifier import JWTAuthVerifier
+    except ImportError:
+        import warnings
+        warnings.warn(
+            'Could not import aiohttp code as aiohttp is not installed.')
+
```

Comment:
`import aiohttp` theoretically may raise `ImportError` when one of its dependencies is not installed. 

`from .auth import JWTAuth` and other imports below it may raise ImportError due to a bug, or when some dependencies are not installed as well.

So, I would do this:

```
    . . .
except ImportError as err:
    warnings.warn(str(err))
```